src/components/BookForm.js

```diff
@@ -50,10 +54,12 @@ class BookForm extends React.Component {
 
   handleSubmit = e => {
     e.preventDefault();
-    this.props.saveBook();
+    const book = this.state.book;
+    this.props.saveBook(book);
   };
 
   handleChange = e => {
```

Comment:
As I mentioned in the channel, it's a good idea to abstract the stateful HTML elements to their own components, so you're not multitasking so much with this onChange handler. It seems a little bloated at first, but it's ultimately more scalable and maintainable for larger apps.

Reply:
Cool, I opened a separate issue #17 to track that